source/_components/aruba.markdown

```diff
@@ -18,28 +18,34 @@ redirect_from:
 
 This platform allows you to detect presence by looking at connected devices to an [Aruba Instant](http://www.arubanetworks.com/products/networking/aruba-instant/) device.
 
+The integration supports both REST API (preferrered) and SSH collection methods.  In order to enable the REST method, you must enable it on your access point:
+
+```
+(Instant AP)# conf t
+(Instant AP)(config)# allow-rest-api
+(Instant AP)(config)# end
+(Instant AP)# commit-apply
+```
+
 Supported devices (tested):
 
 - ARUBA AP-105
-
-<p class='note warning'>
-This device tracker needs telnet to be enabled on the router.
-</p>
```

Comment:
Not sure about this removal?

Doesn't this mean it needs either the telnet or rest enabled?
Are any of those enabled by default?